vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -32,6 +32,14 @@
 
 #define VIC_MPI_ROOT 0
 
+/******************************************************************************
+ *  * @brief   This structure stores netCDF file name and corresponding nc_id
+ *   *****************************************************************************/
+typedef struct {
+    char    nc_filename[MAXSTRING];
+    int     nc_id;
+} nameid_struct;
+
```

Comment:
Well - the `nameid_struct` is not specific to the use of `mpi` so it does not really belong there. It's specific to the use of `netcdf`. Let me think about it.